cfpb/rental-assistance-finder
# Customize react-select styles

Update styling of dropdown to match CFPB design system select pattern.

## Additions

- Add custom styles to filter component

## Testing

1. Check out branch and yarn start
2. Verify dropdown styles

## Screenshots
<img width="739" alt="Screen Shot 2021-08-05 at 12 28 27 PM" src="https://user-images.githubusercontent.com/778171/128409558-4bc7cacf-5ae4-430a-9f9a-2d46e19a3738.png">
